backend/app/alembic/env.py

```diff
@@ -43,7 +44,7 @@ def get_url():
 
 
 def include_object(object, name, type_, reflected, compare_to):
-    if name == "spatial_ref_sys":
+    if name and (name == "spatial_ref_sys" or re.match(r"assignments_.+", name)):
```

Comment:
(pp) add a comment here

Reply:
oop forgot to do this. leaving unresolved so I can get back to it